kore/test/Test/Kore/Step/Substitution.hs

```diff
@@ -133,7 +133,11 @@ test_mergeAndNormalizeSubstitutions =
     , testCase "Double constructor is bottom with variables"
         -- [x=constructor(y)] + [x=constructor(constructor(y))]  === bottom?
         $ do
-            let expect = Left (UnificationError UnsupportedPatterns)
+            let expect = Left $ UnificationError $ UnsupportedPatterns $ unlines
```

Comment:
Could you add helpers to construct the `UnificationError`, that way it's easier to change or add more in the future?

Reply:
Done.